### PR TITLE
Finance message on lower tier registration finance section

### PR DESCRIPTION
**Pull request by cintamani**
This addresses the special case of lower-tier registrations when we have no financial information.
Part of this is also the refactoring of the finance section out of the main views and into a partial and shared presenter.

### CODE REVIEW

File: config/locales/shared/registrations.en.yml [hunk 1]
```diff
@@ -1,6 +1,32 @@
...
+      show:
+        filler: "–"
+        attributes:
+          location:
+            england: "England"
+            wales: "Wales"
+            scotland: "Scotland"
+            northern_ireland: "Northern Ireland"
+            overseas: "Not in the United Kingdom"
+        business_information:
```
Comment (cintamani) [inline config/locales/shared/registrations.en.yml]:
```suggestion
      business_information:
```

File: config/locales/shared/registrations.en.yml [hunk 2]
```diff
@@ -1,6 +1,32 @@
...
+          labels:
```
Comment (cintamani) [inline config/locales/shared/registrations.en.yml]:
```suggestion
        labels:
```

File: config/locales/shared/registrations.en.yml [hunk 3]
```diff
@@ -1,6 +1,32 @@
...
+            location: "Place of business: %{location}"
```
Comment (cintamani) [inline config/locales/shared/registrations.en.yml]:
```suggestion
          location: "Place of business: %{location}"
```

File: config/locales/shared/registrations.en.yml [hunk 4]
```diff
@@ -1,6 +1,32 @@
 en:
   shared:
     registrations:
+      finance_details:
+        finance_information:
+          lower_tier_message: "Lower tier registration - charge are not applicable."
```
Comment (Cruikshanks) [inline config/locales/shared/registrations.en.yml]: Just confirmed this with @glancyea . Just a typo

```suggestion
          lower_tier_message: "Lower tier registration - charges are not applicable."
```

Comment (cintamani) [reply config/locales/shared/registrations.en.yml]: Cheers! :)

**Pull request merged**